### DISCUSSION TITLE
로그 생성(갱신) 주기 관련 질문

### DISCUSSION THREAD

**Discussion by ghost** [luftaquila/monolith#16] [Q&A] [answered]
안녕하세요,

TMA-1 코드를 보다가 SD_SYNC() 함수가 TIMER_1S라는 함수에 있던데, 혹시 로그 생성(갱신) 주기를 1초에 1번이라고 보는 게 타당할까요? 코드를 보다가 넘겨짚는 게 싫어 여쭤봅니다.
오늘 하루도 좋은 하루 되세요. 미리 감사합니다.

//문서에 해당 내용 확인했습니다. 저의 부주의로 확인을 미처 못했네요.. 죄송합니다 해결했습니다!

**Comment by ghost** [comment]
해결했습니다. 감사합니다.

**Reply by luftaquila** [reply, marked as answer]
네 맞습니다 :)

100ms마다 트리거되는 `TIM1`에 의해 제어되고 관련 코드는 `device/TMA-1/Core/Src/tim.c`의 `HAL_TIM_PeriodElapsedCallback()` 에 있습니다.

감사합니다.
